dashboard/src/views/Calculate.vue

```diff
@@ -0,0 +1,267 @@
+<!-- View for workflows presented in usecases 1A and 1B
+-->
+<template>
+  <div class="calculate-performance">
+    <div v-if="errorState">
+      {{ apiErrors }}
+    </div>
+    <template v-if="!systemLoading && !errorState">
+      Calculate Performance For System
+      <b>{{ system.definition.name }}</b>
+      <br />
+      <template v-if="!jobSubmitted">
+        <div class="my-1">
+          <div class="my-1">
+            I want to calculate the system output using...
+            <br />
+            <div class="ml-1 mt-1">
+              <input
+                id="predicted"
+                value="predicted"
+                type="radio"
+                v-model="workflow"
+              />
+              <label for="predicted">
+                weather data provided when the system was designed.
+              </label>
+              <br />
+              <input
+                id="expected"
+                value="expected"
+                type="radio"
+                v-model="workflow"
+              />
+              <label for="expected">
+                actual weather data during system operation.
+              </label>
+              <br />
+            </div>
+          </div>
+          <div class="my-1">
+            My data files includes:
```

Comment:
```suggestion
            My data file includes:
```

I'm assuming a single file is upload?